### PR TITLE
Device support section

### PR DESCRIPTION
**Pull request by estenberg**
Added new Device Support section instead of Device Integration.

Content from Device Integration is now at the bottom of Device Support.

The purpose of this page is to be the starting / link-to page for anything relating to support for hardware or OSes.

### CODE REVIEW

**Comment by estenberg** [comment]
Argh I broke lots of links. Will fix.

File: 02.Overview/02.Device-Support/docs.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+For other types of OSes you can either use a nearby Linux system to update it via a
+[proxy deployment](../../02.Overview/01.Introduction/docs.md#proxy-deployments) or
+[create a custom Mender client](https://hub.mender.io/t/how-to-write-a-custom-client-interfacing-a-mender-server).
+
+
+## Board integration overview
+
+You can customize a an existing board integration or create your own
+to meet your exact needs. A board integion generally consists of the following.
```
Comment (drewmoseley) [inline 02.Overview/02.Device-Support/docs.md:119]: A board integration

**Review by drewmoseley — CHANGES_REQUESTED** [review]

**Review by mirzak — APPROVED** [review]
Looks good (once Drews comment is handled :))

File: 02.Overview/02.Device-Support/docs.md (hunk 1 above)
Comment (estenberg) [reply 02.Overview/02.Device-Support/docs.md]: Thanks, fixed.

**Pull request merged**